python/paddle/distributed/auto_parallel/parallelizer.py

```diff
@@ -190,7 +193,7 @@ def _get_dist_program(self, rank, dist_context=None, relaunch_phase=False):
         # serial forward pass
         self._apply_pre_optimization_passed(completed_main_program,
```

Comment:
Rename `_apply_pre_optimization_passed `to `_apply_pre_optimization_passes` and `_apply_post_optimization_passed` to `_apply_post_optimization_passes`.

Reply:
done.

Reply:
done.